src/__tests__/Carousel.tsx

```diff
@@ -3,8 +3,7 @@ import ReactDOM from 'react-dom';
 import { shallow, mount, ReactWrapper } from 'enzyme';
 import renderer from 'react-test-renderer';
 import * as index from '../index';
-// @ts-ignore
-import Swipe, { ReactEasySwipeProps } from 'react-easy-swipe';
+import Swipe, { SwipeProps as ReactEasySwipeProps } from 'react-easy-swipe';
```

Comment:
The @ts-ignored props were undefined in the react-easy-swipe library, maybe renamed in some version?